oca_dependencies.txt

```diff
@@ -1,4 +1 @@
-# list the OCA project dependencies, one per line
-# add a github url if you need a forked version
-project
```

Comment:
This is why it's failing. You shouldn't remove these lines.

Reply:
Thank you! Change made.